docker-compose.yml

```diff
@@ -3,24 +3,22 @@ version: '3.8'
 services:
   backend_common:
     build: backend_api_honey/
+    container_name: backend
     ports:
       - "8000:8000"
     environment:
-      - DATABASE_URL=postgresql://user:password@db:5432/mydatabase
+      - DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}
     depends_on:
       - db
-
   db:
-    image: postgres:13
+    image: postgres:17
     restart: always
+    container_name: postgres
     environment:
-      POSTGRES_USER: user
-      POSTGRES_PASSWORD: password
-      POSTGRES_DB: mydatabase
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
```

Comment:
use something like this instead
    env_file:
      - storage/secrets/secrets_database.env